packages/typescript-fetch-client2/templates/api.hbs

```diff
@@ -1,17 +1,16 @@
 {{>header}}
 
-import { Configuration } from "./configuration";
-import { BASE_PATH, COLLECTION_FORMATS, encodeURIPathSegment, FetchAPI, FetchArgs, BaseAPI, RequiredError, defaultFetch } from "./runtime";
-import { Api } from "./models";
+import { Configuration } from "../configuration";
+import { BASE_PATH, COLLECTION_FORMATS, encodeURIPathSegment, FetchAPI, FetchArgs, BaseAPI, RequiredError, defaultFetch, dateToString } from "../runtime";
+import { Api } from "../models";
 {{#ifeq dateApproach 'blind-date'}}
 import { LocalDateString, LocalTimeString, LocalDateTimeString, OffsetDateTimeString } from 'blind-date';
 {{/ifeq}}
 {{>hooks/apiImports}}
 
 export type FactoryFunction<T> = (configuration?: Configuration, basePath?: string, fetch?: FetchAPI) => T;
 
-{{#each groups}}
-export namespace {{className name}}Api {
```

Comment:
We're no longer exporting the parameter interfaces?

Reply:
To maintain the same behaviour as before in `apis.ts`, the class and this namespace are both exported in the default export at the bottom